<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QWidget" name="Main">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Hypixel Games Randomizer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/res/icon.jpg</normaloff>:/res/icon.jpg</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>601</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="bedwars">
        <item>
         <widget class="QLabel" name="bwLabel">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Bedwars</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="bwLine">
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="soloBW">
          <property name="text">
           <string>Solo BW</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="teamBW">
          <property name="text">
           <string>Doubles BW</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="threesBW">
          <property name="text">
           <string>3v3v3v3 BW</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="foursBW">
          <property name="text">
           <string>4v4v4v4 BW</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="skywars">
        <item>
         <widget class="QLabel" name="swLabel">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Skywars</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="swLine_2">
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="soloSW">
          <property name="text">
           <string>Solo SW</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="teamSW">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Doubles SW</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="labsoloSW">
          <property name="text">
           <string>Lab Solo SW</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="labteamSW">
          <property name="text">
           <string>Lab Doubles SW</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="tntgames">
        <item>
         <widget class="QLabel" name="tntLabel">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TNT Games</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="tntLine">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="tntrun">
          <property name="text">
           <string>TNT Run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="pvprun">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>PvP Run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="tnttag">
          <property name="text">
           <string>TNT Tag</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="bowspleef">
          <property name="text">
           <string>Bow Spleef</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="others">
        <item>
         <widget class="QLabel" name="othersLabel">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Others</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="othersLine">
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="hypixelsays">
          <property name="text">
           <string>Hypixel Says</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="miniwalls">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Mini Walls</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="partygames">
          <property name="text">
           <string>Party Games</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="solospeedUHC">
          <property name="text">
           <string>Solo Speed UHC</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>75</number>
      </property>
      <property name="rightMargin">
       <number>75</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="buildbattle">
        <item>
         <widget class="QLabel" name="bbLabel">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Build Battle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="bbLine">
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="soloBB">
          <property name="text">
           <string>Solo Build Battle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="teamBB">
          <property name="text">
           <string>Team Build Battle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="gtb">
          <property name="text">
           <string>Guess The Build</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="bbSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>19</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="mudermystery">
        <item>
         <widget class="QLabel" name="mmLabel">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Murder Mystery</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="mmLine">
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="classicMM">
          <property name="text">
           <string>Classic MM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="doubleMM">
          <property name="text">
           <string>Double Up MM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="infection">
          <property name="text">
           <string>Infection</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="mmSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>19</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="duels">
        <item>
         <widget class="QLabel" name="duelsLabel">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Duels</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="duelsLine">
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="uhcdm">
          <property name="text">
           <string>UHC Deathmatch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="bridge2x4">
          <property name="text">
           <string>Bridge 4v4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="bridge4x3">
          <property name="text">
           <string>Bridge 3v3v3v3</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="duelsSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>19</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="randomizeButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Randomize!</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>558</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Roboto Th</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style = &quot;font-family: 'hooge 05_54';&quot;&gt;Hypixel Games Randomizer&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-family:'Roboto Th'; font-size:12pt;&quot;&gt;by Yuushi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>150</number>
      </property>
      <property name="rightMargin">
       <number>160</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Number of players in the party:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="nbPlayers">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <item>
         <property name="text">
          <string>≤ 8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>≤ 10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>≤ 12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>≤ 16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>≤ 24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>≤ 30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>≤ 32</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
